actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -239,6 +239,9 @@ def link_to(name = nil, options = nil, html_options = nil, &block)
       #   be placed
       # * <tt>:params</tt> - \Hash of parameters to be rendered as hidden fields within the form.
       #
+      # +html_options+ not specified here will be passed to the form submit tag element. For example
+      # the submit element class attribute can be set with the <tt>:class</tt> option.
```

Comment:
What if we move this up and combine it with `There are a few special +html_options+:`?

> Most values in +html_options+ are passed through to the button element, but the following are handled separately:

Reply:
Yep, that'd makes sense to me!

Reply:
I'm realizing line https://github.com/rails/rails/pull/41558/files#diff-68404eacab886d3ef2d43242eb40e7e66d4ff6135433ecc8a821faffc6104586R220 has a small mention of the behavior there. Maybe providing the `class` example there would suffice.

Reply:
I think `You can control the form submission and input element behavior using +html_options+.` is meant to lead into the mention of the `:method` method option.  The logical flow of that paragraph feels a bit jumbled already, so if you want to include a `class` example there, I think it would be good to reorganize.

In any case, I do think the change in the `==== Options` section is worthwhile.

Reply:
Hmm that's fair. I see what you mean. So per your original idea, adjust the lead in sentence(s) to the html_options. I think providing an explicit example is really helpful but a bit verbose.

```ruby
      # ==== Options
      # The +options+ hash accepts the same options as +url_for+.
      #
      # +html_options+ listed here will be used to control the form. All others are passed down to
      # the form submit element. For example the <tt>:class</tt> option would set the form submit
      # class attribute.
      # * <tt>:method</tt> - \Symbol of HTTP verb. Supported verbs are <tt>:post</tt>, <tt>:get</tt>,

```

Reply:
I pushed a commit to show what I was thinking.  If it looks good to you, you can [squash the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits), and I'll merge.